pkg/cli/tail.go

```diff
@@ -222,24 +274,51 @@ func (c *TailCommand) Run(ctx context.Context, args []string) error {
 	return nil
 }
 
-func (c *TailCommand) tail(ctx context.Context) ([]*loggingpb.LogEntry, error) {
-	var p logPuller
-	if c.testPuller != nil {
-		p = c.testPuller
-	} else {
-		logClient, err := logging.NewClient(ctx)
+func (c *TailCommand) stream(ctx context.Context, extra []validation.Validator, puller logPuller) error {
+	// logCh is used to receive logEntriesfrom StreamPull
+	logCh := make(chan *loggingpb.LogEntry)
```

Comment:
Should we buffer this channel?

Reply:
I think it's OK, in StreamPull (https://github.com/abcxyz/lumberjack/blob/main/pkg/cloudlogging/puller.go#L148-L153), we send the logEntries one by one, and the for loop here receives them one by one, so I don't think buffer this channel doesn't make the big difference. 
Or is there any other concerns that I didn't get?